media-gfx/gnofract4d/gnofract4d-4.0.1.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1 gnome2-utils xdg-utils
```

Comment:
You should perhaps use the new xdg.eclass instead of gnome-utils and xdg-utils to handle postinst and postrm for you? Then you can also bump EAPI to 7.

(note that with EAPI-7 virtual/pkgconfig is a BDEPEND)

Reply:
xdg is much better, used that instead and deleted postinst and postrm.

EAPI 7 discussed below.